enginetest/queries/script_queries.go

```diff
@@ -1326,6 +1326,26 @@ var ScriptTests = []ScriptTest{
 			},
 		},
 	},
+	{
+		Name: "ALTER TABLE MODIFY column making UNIQUE",
+		SetUpScript: []string{
+			"CREATE table test (pk int primary key, uk int)",
+			"ALTER TABLE `test` MODIFY column uk int unique",
+		},
+		Assertions: []ScriptTestAssertion{
+			{
+				Query: "show create table test",
```

Comment:
Again this might sound a bit orthogonal out of context, but plan tests can sometimes be harder to maintain than correctness tests. A correctness test that verifies uniqueness behavior (eg insert same row twice, check for error on second) might have a longer shelf-life because it's probably rare that we change the error behavior. A show-create might be changed less frequently than an `EXPLAIN plan`, but you could imagine how a slight tweak to how we print `SHOW CREATE` might be painful to fixup.

Reply:
Fixed.